src/flow_solver/flow_solver_cases/periodic_turbulence.cpp

```diff
@@ -734,53 +743,82 @@ void PeriodicTurbulence<dim, nstate>::compute_unsteady_data_and_write_to_table(
+        current_numerical_entropy_change_FRcorrected = (current_numerical_entropy - previous_numerical_entropy + dg->FR_entropy_contribution)/initial_numerical_entropy_abs;
+        this->previous_numerical_entropy = current_numerical_entropy;
+    }
+    this->cumulative_numerical_entropy_change_FRcorrected+=current_numerical_entropy_change_FRcorrected;
 
     if(this->mpi_rank==0) {
         // Add values to data table
         this->add_value_to_data_table(current_time,"time",unsteady_data_table);
-        if(do_calculate_numerical_entropy) this->add_value_to_data_table(numerical_entropy,"numerical_entropy",unsteady_data_table);
-        this->add_value_to_data_table(integrated_kinetic_energy,"kinetic_energy",unsteady_data_table);
-        this->add_value_to_data_table(integrated_enstrophy,"enstrophy",unsteady_data_table);
-        if(is_viscous_flow) this->add_value_to_data_table(vorticity_based_dissipation_rate,"eps_vorticity",unsteady_data_table);
-        this->add_value_to_data_table(pressure_dilatation_based_dissipation_rate,"eps_pressure",unsteady_data_table);
-        if(is_viscous_flow) this->add_value_to_data_table(strain_rate_tensor_based_dissipation_rate,"eps_strain",unsteady_data_table);
-        if(is_viscous_flow) this->add_value_to_data_table(deviatoric_strain_rate_tensor_based_dissipation_rate,"eps_dev_strain",unsteady_data_table);
+        if(do_calculate_numerical_entropy) this->add_value_to_data_table(this->cumulative_numerical_entropy_change_FRcorrected,"numerical_entropy_scaled_cumulative",unsteady_data_table);
+        if(do_calculate_numerical_entropy) this->add_value_to_data_table(dg->entropy_M_norm,"entropy_change_estimate_M_norm",unsteady_data_table);
+        //if(do_calculate_numerical_entropy) this->add_value_to_data_table(current_numerical_entropy_change_FRcorrected,"numerical_entropy_scaled_this_tstep",unsteady_data_table);
+        if(is_rrk) this->add_value_to_data_table(relaxation_parameter, "relaxation_parameter",unsteady_data_table);
+        if (do_calculate_overintegrated_quantities) this->add_value_to_data_table(integrated_kinetic_energy,"kinetic_energy",unsteady_data_table);
+        if (do_calculate_overintegrated_quantities) this->add_value_to_data_table(integrated_enstrophy,"enstrophy",unsteady_data_table);
+        if(is_viscous_flow && do_calculate_overintegrated_quantities) this->add_value_to_data_table(vorticity_based_dissipation_rate,"eps_vorticity",unsteady_data_table);
+        if (do_calculate_overintegrated_quantities) this->add_value_to_data_table(pressure_dilatation_based_dissipation_rate,"eps_pressure",unsteady_data_table);
+        if(is_viscous_flow && do_calculate_overintegrated_quantities) this->add_value_to_data_table(strain_rate_tensor_based_dissipation_rate,"eps_strain",unsteady_data_table);
+        if(is_viscous_flow && do_calculate_overintegrated_quantities) this->add_value_to_data_table(deviatoric_strain_rate_tensor_based_dissipation_rate,"eps_dev_strain",unsteady_data_table);
```

Comment:
i think grouping some of these if statements would make it easier to read

Reply:
Per discussion, I have removed this parameter.